contrib/ci/run

```diff
@@ -19,7 +19,11 @@
 
 set -o nounset -o pipefail -o errexit
 declare -r CI_DIR=`dirname "\`readlink -f \"\$0\"\`"`
-export PATH=$CI_DIR:$PATH
+# sbin is not part of the PATH when running our CI.
+# As some binaries checks depend on having sbin in the
+# PATH, let's add it manually here.
+declare -r SBIN_PATH="/usr/local/sbin:/sbin:/usr/sbin"
+export PATH=$CI_DIR:$SBIN_PATH:$PATH
 export LC_ALL=C
```

Comment:
That will not solve direct invocation of `make intgcheck` without */sbin in PATH. Moreover some distributions
have in daemon binaries in /usr/lib/mit/sbin/. We need to use `pkg-config` and/or `krb5-config` for detection.
And in should be done in configure and not in external scripts.

Reply:
Okay. I'm not going to revisit this PR anytime soon, but your comment does make sense and will be considered whenever I (or someone else) revisit this PR.

Thanks for the review.